resources/maven-project/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```

Comment:
So we will have to keep these urls in sync each time maven releases new versions upstream?

Reply:
this is a fixed URL. When there is a new version of Maven, the old ones stays at their places. So when there is a new version of Maven, we can live with this one as long as we want.

This is generated by Takari Wrapper. We should not maintain this url directly ourself. When there is a new version of Maven, usually, there is a new version of takari and we can call
`mvn -N io.takari:maven:X.X.X:wrapper` in `resources/maven-project` that will take care of updating all of what is needed.
see https://github.com/takari/maven-wrapper#maven-wrapper

Reply:
Cool. So long as they are pretty permanent and have some well-defined ways of updating them down the line, life should be good. Thanks